prow/pod-utilities.md

```diff
@@ -64,3 +83,38 @@ Additional fields may be required for some use cases:
 to clone the repo to a path different than the default of `/go/src/github.com/org/repo/` (e.g. `/go/src/k8s.io/kubernetes/kubernetes`).
 - Jobs that require additional repos to be checked out can arrange for that with
 the `exta_refs` field.
+
+```yaml
+- name: post-job
+  agent: kubernetes
+  decorate: true
+  ssh_key_secrets:
+  - ssh-secret
+  clone_uri: "git@github.com:org/repo.git"
+  extra_refs:
+  - org: org2
+    repo: repo2
```

Comment:
Is this repo because of `repo.git` or because it is always repo?

Reply:
I'm not sure what you are asking here. My other changes may have clarified this though?